synedrion/src/paillier/encryption.rs

```diff
@@ -39,11 +41,28 @@ impl<P: PaillierParams> RandomizerMod<P> {
     }
 
     pub fn pow_signed(&self, exponent: &Signed<P::Uint>) -> Self {
-        Self(self.0.pow_signed(exponent))
+        let abs_exponent = exponent.abs();
```

Comment:
Can we use functions from `uint/pow.rs` here?

Reply:
I did some work on more ergonomic exponentiation [here](https://github.com/entropyxyz/synedrion/pull/145) but didn't want to make this PR noisier than it already is so that PR builds upon this one.